net/ntpd/Makefile

```diff
@@ -146,6 +148,7 @@ define Package/ntp-utils/install
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpdc/ntpdc $(1)/usr/sbin/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ntpq/ntpq $(1)/usr/sbin/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/util/ntptime $(1)/usr/sbin/
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/sntp/sntp $(1)/usr/sbin/
```

Comment:
You need also to check if we have compile sntp. If not then this will fail.

Reply:
Only when ntp-utils are selected, will sntp will be built and installed. See line 112.

@tripolar requested this in this PR above.